www/caddy/src/etc/ssl/ext_sources/caddy.conf

```diff
@@ -0,0 +1,4 @@
+[location]
+base=/var/db/caddy/data/caddy/certificates
+pattern=.*\.crt$
```

Comment:
just for the record this means it's a regex, not a glob pattern. I would rather like see an auto-anchored regex at least than leaving the door open for partial matches on filenames CC @AdSchellevis